packages/devextreme/js/__internal/ui/tree_view/m_tree_view.base.ts

```diff
@@ -801,11 +801,13 @@ const TreeViewBase = (HierarchicalCollectionWidget as any).inherit({
 
   _addContent($container, itemData) {
     const { html, url } = itemData;
-
-    if (url) {
-      $container.html(html);
-      const link = this._getLinkContainer(this._getIconContainer(itemData), this._getTextContainer(itemData), itemData);
-      $container.append(link);
+    if (this.option('adaptivityEnabled')) {
+      if (url) {
+        $container.html(html);
+        // eslint-disable-next-line max-len
```

Comment:
[code style]
don't u think that it's better to refactor than silent lint error? :)

Reply:
[d45533d](https://github.com/DevExpress/DevExtreme/pull/27647/commits/d45533db3d0fdfffa3fcf4d8a704612c6f429a4e)